draft-ietf-mls-extensions.md

```diff
@@ -483,6 +491,99 @@ The `media_type` MAY be zero length, in which case, the media type of the
 `application_content` is interpreted as the first MediaType specified in
 `required_media_types`.
 
+## SelfRemove Proposal
+
+The design of the MLS protocol prevents a member of
+an MLS group from removing itself immediately from the group. To cause
+an immediate change in the group, a member must send a Commit message.
```

Comment:
The first of these three sentences seems to be the conclusion of the second and third. Maybe change up the order a bit?